### ISSUE TITLE
Feature Request - Helm Integration

### ISSUE THREAD

**Issue by mediapathic** [joostkremers/criticmarkup-emacs#12]
I'd love to be able to pull up a helm buffer with a list of changes in the current buffer, and jump between them. My original thought on this was a separate buffer, something like an agenda view with follow-mode. I think overall a helm implementation would be more useful, but the separate buffer might also be useful. 

Essentially, I want a list of changes in the current buffer that I can use to jump between them. 

(Also, thanks so much for giving me CM in emacs!)

**Comment by joostkremers** [comment]
Yes, I can see how this would be useful. I don't use helm myself, so I'm not entirely sure how to go about implementing it. Using swiper (based on ivy) it could be done with a well-crafted regex, but I'm not sure such a thing exists for helm.

It really would have to be a regex, because `cm-mode` doesn't otherwise have any notion of the changes in a buffer...
